ldes-fragmentisers/ldes-fragmentisers-timebased-hierarchical/src/test/resources/example-ldes-member.nq

```diff
@@ -0,0 +1,17 @@
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483> <http://www.w3.org/ns/prov#generatedAtTime> "2020-12-28T09:36:37.127Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483> <http://purl.org/dc/terms/description> "2630 Aartselaar, Baron van Ertbornstraat: Werfzone"^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#langString> .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483> <http://www.w3.org/ns/adms#versionNotes> "MobilityHindranceWasImportedFromLegacy"@nl-be .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483> <http://purl.org/dc/terms/created> "2023-01-01T00:00:00.000Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483> <https://gipod.vlaanderen.be/ns/gipod#gipodId> "10228622"^^<http://www.w3.org/2001/XMLSchema#integer> .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483> <http://purl.org/dc/terms/modified> "2020-12-14T13:48:33.083Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/taxonomies/mobility-hindrance/consequencetypes/e4ea1344-aa27-40e8-b5af-94ec5f7956f8> <http://www.w3.org/2004/02/skos/core#prefLabel> "Geen doorgang voor voetgangers"@nl-be .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/organisations/fedab33f-792a-029c-9b34-9d9cfe7d6245> <http://www.w3.org/2004/02/skos/core#prefLabel> "EagleBe Smartcity"^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#langString> .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/taxonomies/zonetypes/0fb72ef7-6ac9-4a70-b295-a30ea215d250> <http://www.w3.org/2004/02/skos/core#prefLabel> "HinderZone"@nl-be .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/taxonomies/statuses/a411c53e-db33-436a-9bb9-d62d535b661d> <http://www.w3.org/2004/02/skos/core#prefLabel> "Onbekend"@nl-be .
+_:B6d6151daf809cfb8e19b0042133ebf90 <http://data.europa.eu/m8g/endTime> "2021-03-27T15:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
+_:B6d6151daf809cfb8e19b0042133ebf90 <http://data.europa.eu/m8g/startTime> "2021-01-04T06:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/ldes/mobility-hindrances> <https://w3id.org/tree#member> <https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483> .
+<https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://data.vlaanderen.be/ns/mobiliteit#Mobiliteitshinder> .
```

Comment:
Can we please generify this? really want to get rid of GIPOD test data 😄

Reply:
I get that, but as this is only in our unit tests, I don't see that as such a problem